db/schema/transaction.ts

```diff
@@ -22,4 +23,4 @@ const transactionSchema = new mongoose.Schema({
     }
 })
 
-module.exports = transactionSchema;
\ No newline at end of file
+module.exports = transactionSchema;

```

Comment:
idk if there's a preference, do we want to do something like `export default { transactionSchema }` or like `export const transactionSchema`? I'm also fine with this version of exporting but was just wondering.

Reply:
@cinderlee does linter take care of those style considerations? We do have this ticket in backlog to set up in the repo: https://github.com/tally-team/tally-backend/issues/44

If not, maybe we need another ticket that enforces specific code style to be consistent lol.

Reply:
from this stackoverflow maybe? https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-commonjs.md 

though it seems like as another answer we can add node configurations for it

Reply:
Cool, I'll add this link in the ticket.